suite/tests/client-interface/drx-scattergather.c

```diff
@@ -497,24 +512,37 @@ test_avx2_avx512_scatter_gather(void)
 #    endif
 #    ifdef UNIX
 #        ifdef __AVX512F__
-    print("Test restoring the mask register upon a fault\n");
     intercept_signal(SIGSEGV, (handler_3_t)&signal_handler_check_k0, false);
     /* This index will cause a fault. The index number is arbitrary.*/
     test_idx32_vec[9] = 0xefffffff;
+    print("Test restoring the gather scratch mask register upon a fault\n");
     if (SIGSETJMP(mark) == 0)
-        test_avx512_restore_mask_fault(ref_sparse_test_buf, test_idx32_vec);
-    print("Test restoring the mask register upon asynchronous events\n");
+        test_avx512_restore_gather_mask_fault(ref_sparse_test_buf, test_idx32_vec);
+    print("Test restoring the scatter scratch mask register upon a fault\n");
+    if (SIGSETJMP(mark) == 0)
+        test_avx512_restore_scatter_mask_fault(ref_idx32_val32_xmm_ymm_zmm,
+                                               test_idx32_vec, output_sparse_test_buf);
     /* We will get the SIGILL from a ud2 instruction that the client will insert. */
     intercept_signal(SIGILL, (handler_3_t)&signal_handler_check_k0, false);
     /* Restore to a valid value. */
     test_idx32_vec[9] = 0x24;
+    print("Test restoring the gather mask register upon asynchronous events\n");
+    if (SIGSETJMP(mark) == 0)
+        test_avx512_restore_gather_mask_clobber(ref_sparse_test_buf, test_idx32_vec);
+    print("Test restoring the scatter mask register upon asynchronous events\n");
     if (SIGSETJMP(mark) == 0)
-        test_avx512_restore_mask_clobber(ref_sparse_test_buf, test_idx32_vec);
-    print("Test updating the mask register upon asynchronous events\n");
+        test_avx512_restore_scatter_mask_clobber(ref_sparse_test_buf, test_idx32_vec,
```

Comment:
style violation: {}.  I would expect the Vera rules to fail Travis for this.

Reply:
Fixed.